test/comment.spec.js

```diff
@@ -0,0 +1,245 @@
+    chai
+      .request(app)
+      .post(`/api/v1/articles/${5}/comments`)
+      .set('x-access-token', userToken)
+      .send({ comment: 'I am a boy' })
+      .end((err, res) => {
+        const {
+          status,
+          body: { success }
+        } = res;
+        expect(status).to.be.equal(404);
+        expect(success).to.be.equal(false);
+        done(err);
+      });
+  });
+});
+
+describe('Post a comment', () => {
+  before(() => updateVerifiedStatus(validCommentUser.email));
+  it('returns invalid comment body', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/articles/${articleSlug}/comments`)
+      .set('x-access-token', userToken)
+      .send({ comment: '' })
+      .end((err, res) => {
+        const {
+          status,
+          body: { success }
+        } = res;
+        expect(status).to.be.equal(422);
+        expect(success).to.be.equal(false);
+        done(err);
+      });
+  });
+});
+describe('Post a comment', () => {
+  before(() => updateVerifiedStatus(validCommentUser.email));
+  it('returns successfully post comment', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/articles/${articleSlug}/comments`)
+      .set('x-access-token', userToken)
+      .send({ comment: 'I am a boy' })
+      .end((err, res) => {
+        const {
+          status,
+          body: { success, comment }
+        } = res;
+        expect(status).to.be.equal(205);
+        expect(success).to.be.equal(true);
+        commentId = comment.id;
+        done(err);
+      });
+  });
+});
+describe('Get comments', () => {
+  before(() => updateVerifiedStatus(validCommentUser.email));
+  it('returns all comments associated with an article', (done) => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${articleSlug}/comments`)
+      .set('x-access-token', userToken)
+      .end((err, res) => {
+        const {
+          status,
+          body: { success, comments }
+        } = res;
+        expect(status).to.be.equal(205);
+        expect(success).to.be.equal(true);
+        expect(comments).to.be.instanceOf(Array);
+        done(err);
+      });
+  });
+});
+describe('Edit a comment', () => {
+  before(() => updateVerifiedStatus(wrongCommentUser.email));
+  it('returns unauthorized access', (done) => {
+    chai
+      .request(app)
+      .patch(`/api/v1/articles/:slug/comments/${commentId}`)
+      .set('x-access-token', userToken2)
+      .send({ comment: 'I am a girl now' })
+      .end((err, res) => {
+        const {
+          status,
+          body: { success }
+        } = res;
+        expect(status).to.be.equal(401);
+        expect(success).to.be.equal(false);
+        done(err);
+      });
+  });
+});
+describe('Edit a comment', () => {
+  before(() => updateVerifiedStatus(validCommentUser.email));
+  it('returns successfully edit comment', (done) => {
+    chai
+      .request(app)
+      .patch(`/api/v1/articles/:slug/comments/${commentId}`)
```

Comment:
Change to `put`

Reply:
As discussed, patch seems to be the more appropriate method for this. Thanks.